Jasmine/Common/ViewControllers/BaseGameViewControllerDelegate.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+/// Sets the shared functionalities that are applicable across all game view controller delegates.
+/// This protocol will be inherited by all the specialsied game view controller delegates.
+protocol BaseGameViewControllerDelegate {
+
+    // MARK: Score Update
+    /// Redisplay the score displayed on the view controller screen with a new score.
+    ///
+    /// - Parameter newScore: the new score to be redisplayed.
+    func redisplay(_ newScore: Int)
```

Comment:
Probably shouldn't omit the argument label. `redisplay(5)` doesn't tell what it's trying to do. `newScore score: Int` would be better.